docs/guide/contracts-in-action/read-only.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Read Only"
```

Comment:
Let's rename just this title here to `Read-only calls`, which is in fact the header's title of this file, but on the side bar it only shows `"Read Only"`.